packages/web/lib/supabase/server.ts

```
@@ -0,0 +1,38 @@
+import { createClient as createSupabaseClient } from '@supabase/supabase-js'
+import { cookies } from 'next/headers'
+import { Database } from '@/types/database'
+
+export async function createServerClient() {
+  const cookieStore = await cookies()
+
+  return createSupabaseClient<Database>(
+    process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)